bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -73,28 +73,29 @@
       <nav id="nav" class="navigator">
         <div class="container">
           <ul class="menu">
-          {% if release.product == 'Firefox for Android' %}
+          {% if release.product == 'Firefox' and release.channel == 'Release' %}
+            <li class="current">{{ _('Desktop') }}</li>
+          {% else %}
             <li><a href="{{ url('firefox.notes') }}">{{ _('Desktop') }}</a></li>
+          {% endif %}
+          {% if release.product == 'Firefox for Android' and release.channel == 'Release' %}
             <li class="current">{{ _('Android') }}</li>
-            <li><a href="{{ url('firefox.notes', platform='ios') }}">{{ _('iOS') }}</a></li>
-          {% elif release.product == 'Firefox for iOS' %}
-            <li><a href="{{ url('firefox.notes') }}">{{ _('Desktop') }}</a></li>
+          {% else %}
             <li><a href="{{ url('firefox.notes', platform='android') }}">{{ _('Android') }}</a></li>
+          {% endif %}
+          {% if release.product == 'Firefox for iOS' and release.channel == 'Release' %}
             <li class="current">{{ _('iOS') }}</li>
           {% else %}
-            <li class="current">{{ _('Desktop') }}</li>
-            <li><a href="{{ url('firefox.notes', platform='android') }}">{{ _('Android') }}</a></li>
             <li><a href="{{ url('firefox.notes', platform='ios') }}">{{ _('iOS') }}</a></li>
           {% endif %}
-            <li class="submenu-title">
+            <li class="{% if release.channel != 'Release' %}current {% endif %}submenu-title">
               <a href="#" role="button" aria-controls="nav-submenu" aria-expanded="false">{{ _('Other Releases') }}</a>
               <ul class="submenu" id="nav-submenu">
                 <li class="title">{{ _('Recent Desktop Releases') }}</li>
-                <li><a href="{{ url('firefox.notes', channel='beta') }}">{{ _('Latest Beta') }}</a></li>
-                <li><a href="{{ url('firefox.notes', channel='developer') }}">{{ _('Latest Developer Edition') }}</a></li>
+                <li><a href="{{ url('firefox.notes', channel='beta') }}">{{ _('Latest Beta &amp; Developer Edition') }}</a></li>
```

Comment:
Rather than make this a long and awkward link, would it be neater if we kept Beta and Dev Edition as separate links that just pointed to the same URL?

![link](https://cloud.githubusercontent.com/assets/400117/25799336/896da9d4-33dc-11e7-8906-298bb91505dc.jpg)
?